src/main/java/mate/academy/springboot/datajpa/dto/mapper/CategoryMapper.java

```diff
@@ -0,0 +1,26 @@
+package mate.academy.springboot.datajpa.dto.mapper;
+
+import mate.academy.springboot.datajpa.dto.CategoryRequestDto;
+import mate.academy.springboot.datajpa.dto.CategoryResponseDto;
+import mate.academy.springboot.datajpa.model.Category;
+import org.springframework.stereotype.Component;
+
+@Component
+public class CategoryMapper implements DtoRequestMapper<CategoryRequestDto, Category>,
```

Comment:
In mappers overwrite only the methods you use. Also, if you only need one method, then write an appropriate name for the mapper

Reply:
I use both methods, but because of the issues you mention below, It doesn't look like that